### ISSUE TITLE
Side-nav: All children of an active item are bold

### ISSUE THREAD

**Issue by roflcopterDorrie** [govau/uikit#490]
# Bug Report

- [x] I’ve read and understood the [Contributing guidelines](https://github.com/govau/uikit/blob/master/CONTRIBUTING.md) and have done my best effort to follow them.
- [x] I’ve read and agree to the [Code of Conduct](https://github.com/govau/uikit/blob/master/CODE_OF_CONDUCT.md).
- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

All the children under an active item are bold.

## What I expected to happen

Only the current item to be bold. 
I am not sure if this is intended or not, but I would have assumed that only the active item should be bold, not its children as well.

## Reproducing

- Module name: side-nav
- Module version: 2.0.4
- Pancake version: 1.2.4
- Node version: 9.10.1
- npm version: 5.6.0
- Pancake settings in your `package.json`:

```
"pancake": {
    "auto-save": true,
    "plugins": true,
    "ignore": [],
    "css": {
      "minified": true,
      "modules": false,
      "browsers": [
        "last 2 versions",
        "ie 8",
        "ie 9",
        "ie 10"
      ],
      "location": "css/",
      "name": false
    },
    "sass": {
      "modules": false,
      "location": "sass/vendors/",
      "name": "uikit.scss"
    },
    "js": {
      "minified": true,
      "modules": false,
      "location": "pancake/js/",
      "name": "pancake.js"
    },
    "react": {
      "location": false
    },
    "json": {
      "enable": false,
      "location": "pancake/",
      "name": "pancake",
      "content": {
        "name": true,
        "version": true,
        "dependencies": true,
        "path": true,
        "settings": true
      }
    }
  }
```

Steps to reproduce:

1. Apply 'active' class to a list item with children

## Attachments

https://codepen.io/anon/pen/QVNjMy

**Comment by alex-page** [comment]
Thanks @roflcopterDorrie get find. This should be a quick fix, i'll get this done today.

**Comment by alex-page** [comment]
This should be fixed now in version `2.0.5`

**Comment by roflcopterDorrie** [comment]
Thanks mate!
